contrib/ansible/openshift/register.sh

```diff
@@ -58,7 +60,6 @@ while read -u 11 line;do
   remote=$(echo $line | awk -F' ' '{print $1}')
   group=$(echo $line | awk -F' ' '{print $2}')
   index=$(echo $line | awk -F' ' '{print $3}')
```

Comment:
Where do we use `index`?  If we don't need it at this point, can we stop writing it up at line 54?